packages/autocomplete-core/utils.ts

```diff
@@ -191,3 +191,7 @@ export function getHighlightedItem<TItem>({
     source,
   };
 }
+
+export function isOrContainsNode(parent: Node, child: Node) {
```

Comment:
what about something like `equalsOrContains` ?

Reply:
"equals" often refers to a value/content equality while here we're talking about the reference itself.